HighEnergyObsCoreExt.tex

```diff
@@ -237,38 +237,42 @@ \subsection{{\em ev\_xel}}
+The existing attributes {\em em\_min\/} and {\em em\_max\/} that define the coverage of the spectral axis (defined as wavelength expressed in units of m) are not user friendly for \gls{HE} where datasets are generally selected according to an energy range ({\em i.e.\/}, inverse wavelength) in units of eV (or scaled units of eV, for example keV, MeV, GeV, TeV, PeV).  Unlike the radio domain where $\lambda = c/\nu$, where $c$ is an almost universally remembered physical constant, the conversion $\lambda = hc/E$ is not simple for the user to express.  As the spectral range covered by \gls{HE} data is many decades larger than for other wavebands, the accurate numerical representations of typical \gls{HE} spectral ranges as {\em em\_min\/}/{\em em\_max\/} requires quantities with many digits of precision and exponents ranging from $\sim\!10^{-5}$--$10^{-22}$.  Since specification of the spectral range is largely fundamental to data discovery in the \gls{HE} regime, we propose to add attributes {\em energy\_min\/} and {\em energy\_max\/} that specify the minimum and maximum spectral range values in units of eV\null.  Note that the sense of these attributes is {\em opposite\/} that of {\em em\_min\/} and {\em em\_max\/} because of the inverse wavelength relationship between energy and wavelength, so numerical comparisons must be transposed ({\em e.g.\/}, $E>E_{\rm thresh}$ becomes $\lambda<hc/E_{\rm thresh}$).  (An alternate approach would be to add attributes {\em em\_min\_energy\/} and {\em em\_max\_energy\/} that represent the energies corresponding to {\em em\_min\/} and {\em em\_max\/} in units of eV\null.  This is less desirable since queries on an energy would need to be specified as {\em em\_max\_energy\/}${}\leq E <{}${\em em\_min\_energy\/}, which is likely confusing.)
```

Comment:
We had also discussed add standard functions to ADQL that do the conversion, like `WAVELENGTH_TO_ENERGY_EV(em.min)` to help the user in these cases?.  Should that be mentioned here as an alterative (or in addition to) this solution?   

From an implementation standpoint, we might also note that the `energy_min`, `energy_max` columns can also be generated  automatically by the database server, since they are just a simple mathematical transformation of the em_min/max columns, so adding these columns should not require re-ingesting data into existing databases.